new doxygen warnings about unsupported xml/html tags caused by inadvertent use of HTML

@jack-woollen your recent changes introduced the following new doxygen warnings (only one shown here):

```
Generating docs for file binv.f90...
/home/ed/NCEPLIBS-bufr/utils/binv.f90:6: warning: Unsupported xml/html tag <bufrfile> found
/home/ed/NCEPLIBS-bufr/utils/binv.f90:6: warning: Unsupported xml/html tag <bufrfile> found

```

The problem is with this text in binv.f90:

`!> Usage: binv <bufrfile> will print bufrfile inventory by message type.
`

Doxygen is trying to interpret '<bufrfile>' as a HTML tag, which it is not.

To fix, we have to add the backslash, which escapes the angle-brackets and causes doxygen to ignore them, like this:

`!> Usage: binv \<bufrfile\> will print bufrfile inventory by message type.
`

Now doxygen will do the correct thing: print the angle-brackets, but not try to interpret them as HTML.

I will put up a PR with the fixed, but I wanted to explain this to you guys so you will know what to do if you see it again. Soon we will have WARN_AS_ERROR turned on, which would have prevented you from committing these changes.

--- a/utils/binv.f90
+++ b/utils/binv.f90
@@ -3,7 +3,7 @@
 !>
 !> @author J Woollen @date 1994
 
-!> Usage: binv <bufrfile> will print bufrfile inventory by message type.
+!> Usage: binv \<bufrfile\> will print bufrfile inventory by message type.
 !>
 !> @return 0 for success, error message otherwise.
 !>

--- a/utils/cmpbqm.f90
+++ b/utils/cmpbqm.f90
@@ -3,7 +3,7 @@
 !>
 !> @author J Woollen @date 1997
 
-!> Usage: cmpbqm <prepbufrfile> will print prep inventory by variable, report type, and qc mark.
+!> Usage: cmpbqm \<prepbufrfile\> will print prep inventory by variable, report type, and qc mark.
 !>
 !> @return 0 for success, error message otherwise.
 !>

--- a/utils/sinv.f90.in
+++ b/utils/sinv.f90.in
@@ -3,7 +3,7 @@
 !>
 !> @author J Woollen @date 2010
 
-!> Usage: sinv <satbufrfile> will print inventory of satellites by platform and instrument.
+!> Usage: sinv \<satbufrfile\> will print inventory of satellites by platform and instrument.
 !>
 !> @return 0 for success, error message otherwise.
 !>

